www/gatsby-node.js

```diff
@@ -6,6 +6,11 @@ const fs = require(`fs-extra`)
 const slash = require(`slash`)
 const slugify = require(`limax`)
 
+const localPackages = `../packages`
+const localPackagesArr = []
+fs.readdirSync(localPackages).forEach(file => {
```

Comment:
This wasn't finished — the idea here is for packages in this repo to use that README instead of what's grabbed from Algolia. The reason is that the Algolia README often doesn't have good links to images.

So take this `localPackagesArr` and use it to filter out the result of the query below getting all NPM package names.

Reply:
I still don't understand exactly what needs to change on this, do you mean `localPackagesArr` would give the readme data to the `template-docs-packages.js` file so that we don't have to rely on Algolia for the readmes? From what I understand that's what it's doing now. So the data displayed would be from the local folders rather than Algolia's search of npm that struggles with images?

Reply:
Yeah — Algolia (via NPM) has a copy of the READMEs for packages in this repo. We'd prefer to use those as they any referenced images.

Reply:
I came back to this this evening to look into finishing it up, sorry for the confusion on this, I want to make sure I can wrap my head around the actual issue though.

As of now `www/gatsby-node` is creating all the pages for the packages library based off of its own query of `allNpmPackage` (which is just getting all the names to create the pages, when I run that query in GraphiQL I don't see html returned for all the packages, just a handful of them). So to fill in the content on each page the `template-docs-packages.js` has it's own query that's filling the content on the screen from the `markdownRemark` type. Apparently there are packages that have images in their readme and they're causing errors? Do we know what any of those packages are?

Reply:
You should create two template components. One for local packages and the other for remote. They'd share the same components for rendering data but the local package would query the local README while remote packages would query the NPM readme. Make sense?

So when you're creating pages, filter out of the NPM packages anything in `localPackagesArr` and then create pages for `localPackagesArr` w/ the local package template.